templates/docs_temp.html

```diff
@@ -103,11 +103,11 @@ <h5 class="text-2xl leading-tight text-orange"><a href="/doc/contributor-guide/i
       </div>
 
       <div class="order-7 lg:order-6 p-6 pt-0 lg:pt-6 dark:text-white text-slate bg-white rounded-lg rounded-t-none lg:rounded-tl-lg lg:rounded-r-none dark:bg-charcoal dark:bg-neutral-700">
-        <h5 class="text-2xl leading-tight text-orange"><a href="{% url 'review-process' %}" class="link-header">Boost Formal Reviews</a></h5>
+        <h5 class="text-2xl leading-tight text-orange"><a href="/doc/formal-reviews/submissions.html" class="link-header">Boost Formal Reviews</a></h5>
```

Comment:
And in template uses as well

```suggestion
        <h5 class="text-2xl leading-tight text-orange"><a href="{% url 'docs-user-guide' content_path='formal-reviews/submissions.html' %}" class="link-header">Boost Formal Reviews</a></h5>
```